data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json

```diff
@@ -1,7 +1,7 @@
 {
-    "@odata.context" : "<%= basepath %>/$metadata#Systems/Members/$entity",
+    "@odata.context" : "<%= basepath %>/$metadata#ComputerSystemCollection.ComputerSystemCollection",
     "@odata.id": "<%= url %>",
-    "@odata.type": "#ComputerSystem.1.0.0.ComputerSystem",
+    "@odata.type": "#ComputerSystem.v1_0_0.ComputerSystem",
```

Comment:
Further down, can you please fix one more old typo that already existed:
Change the following line to be "Processor" instead of "Processors"
```
@odata.id": "<%= basepath %>/Systems/<%=identifier%>/Processors/<%=i%>"
```